src/components/browserSupport/hooks/useBrowserSupport.js

```diff
@@ -107,7 +108,7 @@ export default (options = {}) => {
       }
     }
 
-    if (isSupported && !isOutdated) {
+    if (isSupported && !isOutdated && !isWebView) {
```

Comment:
this will show usupported everyone in the app (ie after signup), we want the webview warning only on the FV screen.
so specifically the useCameraSupported on FV screen only should use the custom onCheck of userbrowsersupport callback with !isWebview.

Reply:
done